packages/shared-base/src/utils/subscription.ts

```diff
@@ -40,6 +40,25 @@ export function createConstantSubscription<T>(value: T): Subscription<T> {
     }
 }
 
+export function createSubscriptionFromGetterSetter<T>({
```

Comment:
this implementation is wrong. you never call the listeners and it will never trigger react update.